### DISCUSSION TITLE
lualine does not pick up catppuccin

### DISCUSSION THREAD

**Discussion by 0xDmtri** [catppuccin/nvim#468] [Q&A] [answered]
For some reason I cannot make lualine to pickup catppuccin... all other settings work fine.

I read about using after and tried, but didn't help.

<img width="1035" alt="Screenshot_2023-04-28_at_3 40 30_PM" src="https://user-images.githubusercontent.com/33504992/235185870-9676dc32-107c-4e35-86a9-48e156e7338b.png">

**Comment by rewhile** [comment]
You didn't show lualine at all in the screenshot?

**Reply by 0xDmtri** [reply]
> You didn't show lualine at all in the screenshot?
<img width="801" alt="Screenshot 2023-04-29 at 1 36 26 PM" src="https://user-images.githubusercontent.com/33504992/235302856-b01525f0-b0b7-4ee0-96f8-52bd20e767a3.png">

Yeah, sorry.

**Reply by rewhile** [reply]
That looks correct

**Reply by 0xDmtri** [reply]
yeah, and the bar never gets changed to catppuccin (( This is also loaded after theme itself.

**Reply by rewhile** [reply]
Are you talking about the top bar? That isn't neovim, you might want to try [catppuccin/iterm](https://github.com/catppuccin/iterm)

**Reply by 0xDmtri** [reply]
> Are you talking about the top bar? That isn't neovim, you might want to try [catppuccin/iterm](https://github.com/catppuccin/iterm)

nono, talking about lualine

**Reply by rewhile** [reply]
Yeah then that's correct (The Normal bg in the screenshot is `#91B3F6` which is catppuccin color), the bar you see in our preview isn't lualine but feline.

**Reply by 0xDmtri** [reply]
omg, so so sorry, new to nvim... sorry for taking ur time to answer this stupid question lol

**Reply by rewhile** [reply]
Why did you load catppuccin after lualine.nvim?

**Reply by 0xDmtri** [reply, marked as answer]
> Why did you load catppuccin after lualine.nvim?

nah, i removed it long time, after was even not a command in Lazy. I took it somewhere and didnt know it was from Packer
